steps/sofy-upload/0.0.3/step.yml

```diff
@@ -0,0 +1,58 @@
+title: Sofy upload
+summary: |
+  Uploads your build to Sofy.
+description: |
+  Uploads your build to Sofy.
```

Comment:
NOTE: Bitrise Workflow Editor supports markdown formatting, you can use 2 spaces at the end of the line to break line (same applies for the Step Inputs).